api/libretime_api/cli.py

```diff
@@ -7,6 +7,7 @@
 
 def main():
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "libretime_api.settings")
+    # os.environ.setdefault("LIBRETIME_CONFIG_FILEPATH", "/etc/airtime/airtime.conf")
```

Comment:
I wonder what would be more practical. Having a default value here simplify the api cli usage on production, but this implies the configuration will always be loaded from the files, and never from the environment variables.

What could change it that even if a config_file path is provided, we don't fail if that file is missing.

Reply:
I don't quite follow what you mean here?

Reply:
Having the path to the configuration file set by default, will allow the user to run the libretime-api commands without having to specify the path to the configuration file.  This means, instead of writing:
```bash
LIBRETIME_CONFIG_FILEPATH=/etc/airtime/airtime.conf libretime-api migrate

# We could simply have
libretime-api migrate
```
The current implementation fails if a filepath to a config file is provided but the file doesn't exist.

So I wonder if we should allow for missing config file even if a file path is provided (using a default or a user provided file path) so that we have the convenience of not having to write the config file path in the CLI and allow users to only use environment variables.

Reply:
Also, since we want to have the stream settings in the configuration file, I don't know how one could cleanly configure stream outputs using environment variables, as they are quiet complex.

I am unsure about our goal regarding environment variable configuration.

Currently we can configure LitreTime using both methods, but we might need to force having a configuration file, even if it will only be used for the stream settings.

Reply:
Yes, I think having a sensible default is a good idea.

Being able to configure things through environment variables is important for containerized applications. We can use [pyaml_env](https://github.com/mkaranasou/pyaml_env) to read environment into the config file. That would just require we have a straightforward way of translating a config key into an env var. For example `a.b.c` could be `A_B_C`. Lists like  `a.b[c]` can become `A_B_1_C`